test/correctness/simd_op_check.cpp

```diff
@@ -1889,6 +1889,9 @@ struct Test {
         check("v*:*.w = vtmpy(v*:*.h, r*.b)", hvx_width/2, i32(in_i16(x - 1))*2 + i32(in_i16(x)) + i32(in_i16(x + 1)));
         check("v*:*.w = vtmpy(v*:*.h, r*.b)", hvx_width/2, i32(in_i16(x - 1)) + i32(in_i16(x)) + i32(in_i16(x + 1)));
 
+        check("v*:*.w = vrmpy(v*:*.ub, r*.b, #*)", hvx_width/1, 2*i32(in_u8(x - 1)) + 3*i32(in_u8(x)) + 5*i32(in_u8(x + 1)) + 7*i32(in_u8(x + 2)));
+        check("v*:*.w = vrmpy(v*:*.ub, r*.b, #*)", hvx_width/1, 9*i32(in_u8(x - 1)) + 8*i32(in_u8(x)) + 7*i32(in_u8(x + 1)));
```

Comment:
Ah of course, I see. At first I thought that was the multiply operand, now I see that it's the control flag.